SDDM 0.16.0 fails to start with consolekit2 because it assumes consolekit2
has the same interface names as logind.

This patch forces sddm to use legacy behaviour when consolekit2 is detected.

Bug: https://bugs.gentoo.org/633920
Bug: https://github.com/sddm/sddm/issues/903
Bug: https://github.com/ConsoleKit2/ConsoleKit2/issues/99

--- a/src/daemon/LogindDBusTypes.cpp
+++ b/src/daemon/LogindDBusTypes.cpp
@@ -60,7 +60,7 @@
 
     if (QDBusConnection::systemBus().interface()->isServiceRegistered(QStringLiteral("org.freedesktop.ConsoleKit"))) {
         qDebug() << "Console kit interface found";
-        available = true;
+        available = false;
         serviceName = QStringLiteral("org.freedesktop.ConsoleKit");
         managerPath = QStringLiteral("/org/freedesktop/ConsoleKit/Manager");
         managerIfaceName = QStringLiteral("/org.freedesktop.ConsoleKit.Manager"); //note this doesn't match logind
